src/bin/tsservices_dev.js

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line node/no-unpublished-import
+import * as ts from "typescript";
+import path from 'path';
+import fs from 'fs';
+
+const LIBDTS = `// Standard types
+interface Date {}
+
+type Record<K extends keyof any, T> = {
+	[P in K]: T;
+};
+`;
+
+const servicesHost = files => ({
```

Comment:
I'd import `servicesHost` from `src/node_modules/app/tsservices/index.js` to avoid duplication